src/main/java/net/rithms/riot/api/RiotApi.java

```diff
@@ -61,35 +61,8 @@
 import net.rithms.riot.api.endpoints.static_data.constant.RuneTags;
 import net.rithms.riot.api.endpoints.static_data.constant.SpellListTags;
 import net.rithms.riot.api.endpoints.static_data.constant.SpellTags;
-import net.rithms.riot.api.endpoints.static_data.dto.Item;
-import net.rithms.riot.api.endpoints.static_data.dto.ItemList;
-import net.rithms.riot.api.endpoints.static_data.dto.LanguageStrings;
-import net.rithms.riot.api.endpoints.static_data.dto.MapData;
-import net.rithms.riot.api.endpoints.static_data.dto.Mastery;
-import net.rithms.riot.api.endpoints.static_data.dto.MasteryList;
-import net.rithms.riot.api.endpoints.static_data.dto.ProfileIconData;
-import net.rithms.riot.api.endpoints.static_data.dto.Realm;
-import net.rithms.riot.api.endpoints.static_data.dto.Rune;
-import net.rithms.riot.api.endpoints.static_data.dto.RuneList;
-import net.rithms.riot.api.endpoints.static_data.dto.SummonerSpell;
-import net.rithms.riot.api.endpoints.static_data.dto.SummonerSpellList;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataChampion;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataChampionList;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataItem;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataItemList;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataLanguageStrings;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataLanguages;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataMaps;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataMastery;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataMasteryList;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataProfileIcons;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataRealm;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataRune;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataRuneList;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataSummonerSpell;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataSummonerSpellList;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataTarballLinks;
-import net.rithms.riot.api.endpoints.static_data.methods.GetDataVersions;
+import net.rithms.riot.api.endpoints.static_data.dto.*;
```

Comment:
Leave these as individual imports. It clarifies which packages classes are coming from.